### ISSUE TITLE
Always gives the time since last boot

### ISSUE THREAD

**Issue by George-Miao** [metrics-rs/quanta#46]
I don't if I'm misunderstanding, `Instant::now().as_unix_duration()` should return a `Duration` since UNIX epoch right? But it always gives me the time since my last boot.

Code:
```rust
use quanta::Instant;
fn main() {
    println!("{:#?}", Instant::now().as_unix_duration());
}
```
Result:
![image](https://user-images.githubusercontent.com/38549822/120944209-d4c16300-c765-11eb-8d19-e3a1c5fcec6d.png)

Environment:
rustc 1.49.0 (e1884a8e3 2020-12-29)
cargo 1.49.0 (d00d64df9 2020-12-05)
5.12.8-arch1-1 ArchLinux
AMD 5950x

**Comment by tobz** [comment]
You're definitely not misunderstanding.... I just have no idea what the hell I was thinking when I added that convenience method. 😂 

Thanks for filing an issue on this!

**Comment by tobz** [comment]
Released a new version of quanta -- `0.8.0` -- that removes the aforementioned method as it didn't work and had no hope of correctly working while still providing performant access.
